src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -2313,9 +2313,11 @@ namespace cryptonote
     {
       std::vector<std::string> args;
```

Comment:
We want to make the user work with an interface similar to prepare registration where they can specify the inputs of each step programatically

```c++
String operator_cut // Double between [0, 1] that is the percentage cut to take from each participant.
String[] addresses // Array of wallet addresses to split the reward with.

// A portion is a double represented as as string between [0..1] inclusiv
// The proportion of the reward that each participant in the address receives.
// The sum of the proportion can be at most 1, but does not need to equal 1 if the pool is open.
// or if loki, then the staking_requirement/loki is the percentage of reward etc etc.
String[] portions/loki

Int initial_contribution // The amount of loki the operator is contributing
Bool make_friendly // If true the registration command is annotated with help about how to use the command.
```

Just for example, that was how the call was structured in the beginning see the git blame for more context.